tests/integration/test_storage.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `ebr_trackerbot` package."""
+
+import sys
+import os
+
+sys.path.append("ebr_trackerbot")
```

Comment:
Why is this needed? During a test run the module get's installed

Reply:
I rewrote the way how to import modules